src/main/java/seedu/address/model/person/Person.java

```diff
@@ -130,6 +139,38 @@ public List<Interaction> addInteractions(List<Interaction> interactions) {
         return this.interactions;
     }
 
+    public Optional<LocalDate> getFollowUpDate() {
+        if (interactions.isEmpty() || lead == null) {
+            return Optional.empty();
+        }
+        LocalDate latestInteractionDate = interactions.get(interactions.size() - 1).getDate();
```

Comment:
Can create a private method called getLastInteraction inside ReminderList and deal with the problem there

Reply:
Yeap will do!